### ISSUE TITLE
[BFW-7591][FEATURE REQUEST] XL Multi Tool - Menu option to unload all filament

### ISSUE THREAD

**Issue by whirlyknurly** [prusa3d/Prusa-Firmware-Buddy#4092]
### Printer model

XL Multi Tool

### Describe the feature

I don't leave filament on my XL between prints, and currently it takes quite a lot of clicking and scrolling around in the "Change Filament in All Tools" menu to set every tool to unload, and I have to do it fairly often, so I would like an option in the Filament menu that just unloads every loaded tool.

**Comment by mhornsby** [comment]
Yes please !.I don't want to stand at the printer unload each tool head one by one. Would like to select unload all , walk away for a few miniutes and come back and pull the filament out.
